src/CMakeLists.txt

```diff
@@ -186,6 +186,14 @@ foreach (plugin_dir ${BUILTIN_PLUGIN_LIST})
     add_subdirectory(${plugin_dir} ${CMAKE_CURRENT_BINARY_DIR}/plugins/${plugin_name})
 endforeach ()
 
+# Write the preload Zeek script for static plugins.
+file(GLOB STATIC_PLUGIN_PRELOADS CONFIGURE_DEPENDS "${CMAKE_BINARY_DIR}/scripts/plugins/*/__preload__.zeek")
+file(WRITE ${CMAKE_BINARY_DIR}/scripts/plugins/__preload__.zeek "# Warning, this is an autogenerated file!\n")
+foreach (static_preload_script ${STATIC_PLUGIN_PRELOADS})
+    string(REPLACE ${CMAKE_BINARY_DIR}/scripts/ "" load_script ${static_preload_script})
+    file(APPEND ${CMAKE_BINARY_DIR}/scripts/plugins/__preload__.zeek "@load ${load_script}\n")
+endforeach ()
+
```

Comment:
Does this fully-overwrite this file every time CMake runs? Say, if you switch branches or update a submodule?

Reply:
There is a line just above that does WRITE which truncates the file to place the line that says the file is autogenerated.